llvm/tools/llvm-exegesis/llvm-exegesis.cpp

```diff
@@ -418,8 +423,15 @@ static void runBenchmarkConfigurations(
         std::optional<StringRef> DumpFile;
         if (DumpObjectToDisk.getNumOccurrences())
           DumpFile = DumpObjectToDisk;
+        std::optional<int> BenchmarkCPU = std::nullopt;
```

Comment:
After the suggestion above, this can be:
`const std::optional<int> BenchmarkCPU = BenchmarkProcessCPU != -1 ? std::nullopt : std::make_optional(BenchmarkProcessCPU);`